modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc 
+
+:_content-type: CONCEPT
+[id="rosa-hcp-classic-comparison_{context}"]
+= Comparing ROSA Classic and hosted control planes for ROSA
+
+{hcp-first} for {product-title} (ROSA) offers a different way to create a managed ROSA cluster. {hcp-first} for ROSA offers a reduced-cost solution with focuses on reliability and efficiency. With a focus on simplicity, you quickly create a new cluster and deploy applications. 
+
+{hcp-first} for ROSA uses only one cluster and two nodes, making it ideal for smaller projects.
+
+.ROSA versions comparison table
+
+[cols="3a,8a,8a",options="header"]
+|===
+| {nbsp} +
+| Hosted Control Plane 
+| Classic
+
+| *What are each of the installation paths?*
+| This installation path uses control plane components, such as etcd, API server, oauth, that are hosted on AWS in a Red Hat-owned and managed OpenShift cluster. 
+| This installation path contains the control plane and infrastructure & worker nodes that all live in customer’s AWS account.
+
+| *Provisioning Time*
+| Approximately 10 minutes 
+| Approximately 40 minutes 
+
+| *Architecture*
+|
+    * Users have no direct access to underlying control plane infrastructure except by using exposed endpoints 
+    * Red Hat's AWS regional infrastructure consists of
+    * One set of clusters responsible for hosting customer control plane components
+    * Another set of clusters responsible for hosting orchestration workloads (ACM)
+|
+    * You are responsible for control plane, infrastructure, and networking
+    * All-in-one {product-title} infrastructure architecture
+    * Red Hat maintains the orchestration workloads (OCM and Hive) in our AWS account
+
+| *Footprint*
+| 1 cluster requires a minimum of 2 nodes
+| 1 cluster requires minimum of 7 nodes
+
+| *Deployment* 
+| 
+    * Deploy using ROSA CLI or web UI
+    * Customers provision "Hosted Clusters" that deploy the control plane components into Red Hat's Management clusters
+    * Customers request "Machine Pools" that deploy worker nodes into the customer's AWS account
+|
+    * Deploy using ROSA CLI or web UI
+    * Worker nodes are deployed as machine pools (vs node pools) with a focus on regional resiliency
+
+| *Upgrades*
+| Selectively upgrade control plane and worker nodes separately
+| Entire cluster is upgraded at one time
+
+| *Regional Availability* 
+| Initially 6 regions available:
+    * us-east-1
+    * us-east-2
+    * us-west2
+    * eu-west-1
+    * eu-central-1
+    * ap-southeast-2
```

Comment:
It only supports this region on production. @arendej would you help confirm it?
./rosa list region --hosted-cp
ID           NAME                    MULTI-AZ SUPPORT    HOSTED-CP SUPPORT
eu-west-1    EU, Ireland             true                true
us-east-1    US East, N. Virginia    true                true

Reply:
This is our "GA" list, not the _current_ list